src/apps/gearbox/ethereum/gearbox.credit-accounts.contract-position-fetcher.ts

```diff
@@ -0,0 +1,127 @@
+import { Inject } from '@nestjs/common';
+import { BigNumberish } from 'ethers';
+import _ from 'lodash';
+
+import { IAppToolkit, APP_TOOLKIT } from '~app-toolkit/app-toolkit.interface';
+import { PositionTemplate } from '~app-toolkit/decorators/position-template.decorator';
+import { DefaultDataProps } from '~position/display.interface';
+import { MetaType } from '~position/position.interface';
+import { ContractPositionTemplatePositionFetcher } from '~position/template/contract-position.template.position-fetcher';
+import {
+  GetDefinitionsParams,
+  DefaultContractPositionDefinition,
+  GetTokenDefinitionsParams,
+  UnderlyingTokenDefinition,
+  GetDisplayPropsParams,
+  GetTokenBalancesParams,
+} from '~position/template/contract-position.template.types';
+import { Network } from '~types/network.interface';
+
+import { CreditManagerV2, GearboxContractFactory } from '../contracts';
+
+const network = Network.ETHEREUM_MAINNET;
+
+// @Register.ContractPositionFetcher({ appId, groupId, network })
+@PositionTemplate()
```

Comment:
When using the decorator @PositionTemplate, you don't need to re-define `appId`, `groupId`, `network`. You can also remove the comment 😉